config/kms/config.go

```diff
@@ -42,24 +39,4 @@ func Configure(p *config.Provider) {
 	p.AddResourceConfigurator("google_kms_secret_ciphertext", func(r *config.Resource) {
 		r.TerraformResource.Schema["plaintext"].Sensitive = false
 	})
-
-	p.AddResourceConfigurator("google_kms_crypto_key_version", func(r *config.Resource) {
```

Comment:
nit: We may consider removing this configurator function because it was only responsible for syncing the schemas.